nixos/modules/services/web-servers/h2o/vhost-options.nix

```diff
@@ -37,6 +37,15 @@ in
       '';
     };
 
+    host = mkOption {
+      type = types.nullOr types.nonEmptyStr;
+      default = null;
+      example = "127.0.0.1";
```

Comment:
Wouldn't a good example be something that sets this to the non-default, like listening on all addresses? `::`, or however the syntax for this is?

Reply:
Ok. So maybe add a line to the description ("If unset, the default (listen on all addresses) applies."